src/application.coffee

```diff
@@ -36,6 +36,9 @@ logTypes =
 	downloadApp:
 		eventName: 'Application download'
 		humanName: 'Downloading application'
+	downloadAppDelta:
+		eventName: 'Application delta download'
```

Comment:
Yeah, the problem is that we only send the event, not the supervisor version, so unless we do a bunch of work external to mixpanel matching up supervisor versions to uuids to events (and if there's a supervisor/host os update track when that was) then we cannot be sure which type of "application download" event we're seeing